src/fullcalendar/eventSources/eventSourceFunction.js

```diff
@@ -32,16 +34,46 @@ export function eventSourceFunction(calendarObjects, calendar, start, end, timez
 			logger.error(error.message)
 			continue
 		}
-
 		for (const object of allObjectsInTimeRange) {
 			const classNames = []
+			let didEveryoneDecline = false
 
-			if (object.status === 'CANCELLED') {
-				classNames.push('fc-event-nc-cancelled')
-			} else if (object.status === 'TENTATIVE') {
-				classNames.push('fc-event-nc-tentative')
+			// You are an organizer
+			if (object.getFirstPropertyFirstValue('ORGANIZER') === `mailto:${principalsStore.getCurrentUserPrincipalEmail}`) {
+				// Check if all the attendees have declined the event
+				if (object.hasProperty('ATTENDEE')) {
+					didEveryoneDecline = true
+					for (const attendeeProperty of object.getPropertyIterator('ATTENDEE')) {
+						const hasDeclined = attendeeProperty.participationStatus === 'DECLINED'
+						if (!hasDeclined) {
+							didEveryoneDecline = false
+						}
+					}
+					if (didEveryoneDecline) {
+						classNames.push('fc-event-nc-all-declined')
+					}
+				}
+				if (object.status === 'CANCELLED') {
+					classNames.push('fc-event-nc-cancelled')
+				} else if (object.status === 'TENTATIVE') {
+					classNames.push('fc-event-nc-tentative')
+				}
```

Comment:
This is a regression. The cancelled and tentative states are global for the event and not tied to organized event. You can set the status on events that have no organizer and attendees.

I think you should move the block outside of the `if ('ORGANIZER')` again.

The frontend tests also fail because of this.

Reply:
Can be reproduced by creating an event and setting the status to cancelled.

**Before:** Background is slightly transparent and the title is crossed.
**Here:** Event is shown as a regular without any styling changes.

Reply:
Thanks for noticing!